components/InputText.tsx

```diff
@@ -0,0 +1,39 @@
+import { Controller, Control } from 'react-hook-form';
+import TextField from '@mui/material/TextField';
+import FormHelperText from '@mui/material/FormHelperText';
+
+type InputProps = {
+  name: string;
+  label: string;
+  control: Control<any>;
+  multiline?: boolean;
+  rows?: number;
+  required?: boolean;
+};
+
+const InputText = ({ name, label, control, multiline, rows, required }: InputProps) => {
+  return (
+    <Controller
+      name={name as any}
+      control={control}
+      rules={{ required: `${label} is required`, validate: value => value.trim() !== '' || `${label} should not be empty` }}
```

Comment:
As we discussed earlier, `rules` value should be dynamic here. I should be able to pass whatever rules I want into this component.